src/main/java/net/dv8tion/jda/internal/handle/ThreadCreateHandler.java

```diff
@@ -35,9 +36,14 @@ protected Long handleInternally(DataObject content)
         if (api.getGuildSetupController().isLocked(guildId))
             return guildId;
 
+        if (api.getThreadChannelById(content.getUnsignedLong("id")) != null)
+            return null;
```

Comment:
When would this happen?

Reply:
Without wanting to test every edge case, I'm assuming the event also fires in some cases when the member is added despite already knowing it exists (moderator permissions). I just want to account for any unexpected situations this way.

Reply:
Lets drop a comment on there indicating that this is guess-based protection instead of specifically known situation based protection